src/express.re

```diff
@@ -268,7 +268,10 @@ module App = {
   external asMiddleware : t => Middleware.t = "%identity";
   /** [asMiddleware app] casts an App instance to a Middleware type */
 
-  external listen : t => port::int => unit = "" [@@bs.send];
+  external listen_ : t => port::int => onListen::(Js.Null_undefined.t Js.Exn.t => unit) [@bs.uncurry] => unit = "" [@@bs.send];
```

Comment:
not as part of this, but we should hide this in a .rei I guess, since it doesn't need to be public.